UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs

```diff
@@ -125,7 +129,7 @@ public async Task<bool> CreateCatalogFile(string batchId, string exchangeSetRoot
             else
             {
                 logger.LogError(EventIds.CatalogueFileIsNotCreated.ToEventId(), "Error in creating catalogue.031 file for BatchId:{BatchId} and _X-Correlation-ID:{CorrelationId}", batchId, correlationId);
-                return false;
+                throw new FulfilmentException(EventIds.CatalogueFileIsNotCreated.ToEventId());
```

Comment:
Kindly change eventid to CatalogFileIsNotCreated. Also correct filename in logerror to catalog.031 file

Reply:
Resolved